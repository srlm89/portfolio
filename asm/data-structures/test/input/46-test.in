ERASE actual/46-test.out
INSERT 32 0.701286 Dragon Wars   720p
PRINT_IN_ORDER actual/46-test.out
INSERT 15 3.570854 James Bond 11 - MoonRaker [1979]
INSERT 33 1.895006 Star Trek [2009] [1080p]
REMOVE_ID 32
PRINT_IN_ORDER actual/46-test.out
INSERT 11 0.245685 Madagascar 2 Escape 2 Africa [2008] [720p]
INSERT 14 7.053906 Die Hard 3   720p
INSERT 10 0.532156 Burn E [720p]
REMOVE_ID 10
PRINT_IN_ORDER actual/46-test.out
INSERT 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p]
INSERT 38 2.609277 Finding Nemo [2003] [720p]
INSERT 12 5.427464 Equilibrium 720p
INSERT 36 1.267181 Aladdin [1992] [720p]
REMOVE_ID 11
REMOVE_ID 36
PRINT_IN_ORDER actual/46-test.out
INSERT 55 4.885959 Final Fantasy The Spirits Within    720p
INSERT 5 9.728215 Body of Lies   720p
INSERT 54 8.624808 Connected   720p
INSERT 25 3.909678 Knowing 2009 720p
INSERT 34 0.925188 Rambo 4 [2008] [720p]
REMOVE_ID 14
REMOVE_ID 55
PRINT_IN_ORDER actual/46-test.out
INSERT 13 2.818495 Doomsday 720p
INSERT 35 7.705054 Ice Age 1 [2002] [720p]
INSERT 19 1.551491 Bad Boy I [1995] [720p]
INSERT 44 7.472310 Duplicity 2009   720p
INSERT 58 2.094819 21   1080p
INSERT 28 1.679300 Ice Age 3 Dawn of the Dinosaurs [2009] [720p]
REMOVE_ID 34
REMOVE_ID 15
REMOVE_ID 12
PRINT_IN_ORDER actual/46-test.out
INSERT 66 4.199150 Eagle Eye    1080p
INSERT 3 3.531948 Horton Hears a Who! [2008] [720p]
INSERT 41 3.812025 Chicken Run [2000] [720p]
INSERT 48 2.112130 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D
INSERT 24 5.036194 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p]
INSERT 69 5.218473 Terminator 2 Judgement Day Directors Cut [1991]   1080p
REMOVE_ID 38
REMOVE_ID 58
REMOVE_ID 66
PRINT_IN_ORDER actual/46-test.out
INSERT 21 5.124632 Monsters vs Aliens [2009] [720p]
INSERT 16 3.041106 iRobot  720p
INSERT 29 6.672017 Star Trek Collection 1-9    720p
INSERT 47 0.322202 Chicken Little 1080p
INSERT 30 3.660891 James Bond 03 - GoldFinger [1964]
INSERT 78 8.245003 007 Licence to Kill [1989] [BluRay] [720p] [DTS]
INSERT 59 7.119242 Jurassic Park 1-3  1080p
INSERT 49 4.041338 Dragonball Evolution [2009] [720p]
REMOVE_ID 25
REMOVE_ID 35
REMOVE_ID 48
REMOVE_ID 19
PRINT_IN_ORDER actual/46-test.out
INSERT 42 1.063909 007 Series
INSERT 65 1.526949 IMAX Ring of Fire [720p] [Documenter]
INSERT 31 5.085944 Big Nothing   720p
INSERT 60 5.062544 The Matrix   1080p
INSERT 17 4.302033 Crank 2, High Voltage   1080p
INSERT 26 1.446125 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 85 5.971929 The Chronicle of Riddick 720p
INSERT 80 8.351246 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs
INSERT 71 0.744343 The Usual Suspect    720p
REMOVE_ID 65
REMOVE_ID 17
REMOVE_ID 26
REMOVE_ID 16
PRINT_IN_ORDER actual/46-test.out
INSERT 7 9.025733 X-Men Origin - Wolverine [2009] [1080p]
INSERT 57 6.903327 Burn E [720p]
INSERT 20 6.324520 Underworld Series
INSERT 89 4.982187 10,000 BC 720p
INSERT 83 9.128202 Punisher 2   720p
INSERT 23 4.299087 Rocky 3 [1982] [BluRay] [720p] [AC3]
INSERT 70 2.588273 Battle for Terra [2007] [1080p]
INSERT 75 8.039718 Big Nothing   720p
INSERT 56 5.219541 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN]
INSERT 94 4.089340 Night At The Museum 2 [720p]
REMOVE_ID 89
REMOVE_ID 52
REMOVE_ID 24
REMOVE_ID 83
REMOVE_ID 69
PRINT_IN_ORDER actual/46-test.out
INSERT 81 2.780032 City of Embers    720p
INSERT 91 2.122059 American Psycho   720p
INSERT 93 5.511501 Hellboy 2     1080p
INSERT 46 1.234344 Die Hard 3   720p
INSERT 92 0.945333 Sixth Sense   720p
INSERT 0 9.258617 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS]
INSERT 82 5.081608 Bridge To Terabithia 720P
INSERT 84 6.959754 Terminator 3 Rise Of The Machine [2003]   720p
INSERT 53 8.721944 A Bug's Life [720p]
INSERT 110 4.048022 The Matrix Revolution   1080p
INSERT 103 5.516839 Hulk 2 - The Incredible Hulk - 720p
REMOVE_ID 0
REMOVE_ID 57
REMOVE_ID 7
REMOVE_ID 13
REMOVE_ID 28
PRINT_IN_ORDER actual/46-test.out
INSERT 77 6.366110 Erin Brockovich 2000.720p
INSERT 43 8.504705 Big Buck Bunny [1080p]
INSERT 101 4.885701 Deep Sea
INSERT 88 4.953958 Jeniffer's Body ( 2009)    1080p
INSERT 27 2.160895 Orphan (2009)    1080p
INSERT 96 9.052782 Final Fantasy The Spirits Within    720p
INSERT 104 0.841493 Taken 1080p
INSERT 64 2.923004 Hulk 2 - The Incredible Hulk - 720p
INSERT 76 3.351120 The Day After Tomorrow 1080p
INSERT 2 9.730302 007 Thunderball [1965]  [BluRay] [720p] [DTS]
INSERT 63 1.817112 Aeon Flux   720p
INSERT 37 4.180401 Finding Nemo [2003] [720p]
REMOVE_ID 78
REMOVE_ID 37
REMOVE_ID 30
REMOVE_ID 43
REMOVE_ID 77
REMOVE_ID 101
PRINT_IN_ORDER actual/46-test.out
INSERT 67 8.151021 IMAX Ring of Fire [720p] [Documenter]
INSERT 95 9.559426 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p
INSERT 112 7.376703 Rocky 5 [1990] [BluRay] [720p] [AC3]
INSERT 124 6.319414 Iron Man    1080p
INSERT 22 4.446526 X-Men 3   1080p
INSERT 99 7.994218 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 50 8.118501 Indiana.Jones.and.the.Kingdom.of.the.Crystal.Skull    1080p
INSERT 51 4.549279 Lion King, The 720p
INSERT 86 4.615148 Kung Fu Panda (BluRay 720p)
INSERT 117 4.067580 James Bond 08 - Live and Let Die [1973]
INSERT 61 1.298710 Eragon [2006] [720p]
INSERT 121 0.971639 Eragon 720p
INSERT 98 3.501873 Madagascar 1 [2005] [1080p]
REMOVE_ID 56
REMOVE_ID 91
REMOVE_ID 29
REMOVE_ID 54
REMOVE_ID 33
REMOVE_ID 31
PRINT_IN_ORDER actual/46-test.out
