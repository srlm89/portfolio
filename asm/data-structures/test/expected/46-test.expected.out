[ 32 0.701286 Dragon Wars   720p ]
[ 15 3.570854 James Bond 11 - MoonRaker [1979] ][ 33 1.895006 Star Trek [2009] [1080p] ]
[ 11 0.245685 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 14 7.053906 Die Hard 3   720p ][ 15 3.570854 James Bond 11 - MoonRaker [1979] ][ 33 1.895006 Star Trek [2009] [1080p] ]
[ 12 5.427464 Equilibrium 720p ][ 14 7.053906 Die Hard 3   720p ][ 15 3.570854 James Bond 11 - MoonRaker [1979] ][ 33 1.895006 Star Trek [2009] [1080p] ][ 38 2.609277 Finding Nemo [2003] [720p] ][ 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ]
[ 5 9.728215 Body of Lies   720p ][ 12 5.427464 Equilibrium 720p ][ 15 3.570854 James Bond 11 - MoonRaker [1979] ][ 25 3.909678 Knowing 2009 720p ][ 33 1.895006 Star Trek [2009] [1080p] ][ 34 0.925188 Rambo 4 [2008] [720p] ][ 38 2.609277 Finding Nemo [2003] [720p] ][ 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 54 8.624808 Connected   720p ]
[ 5 9.728215 Body of Lies   720p ][ 13 2.818495 Doomsday 720p ][ 19 1.551491 Bad Boy I [1995] [720p] ][ 25 3.909678 Knowing 2009 720p ][ 28 1.679300 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 33 1.895006 Star Trek [2009] [1080p] ][ 35 7.705054 Ice Age 1 [2002] [720p] ][ 38 2.609277 Finding Nemo [2003] [720p] ][ 44 7.472310 Duplicity 2009   720p ][ 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 54 8.624808 Connected   720p ][ 58 2.094819 21   1080p ]
[ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 13 2.818495 Doomsday 720p ][ 19 1.551491 Bad Boy I [1995] [720p] ][ 24 5.036194 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 25 3.909678 Knowing 2009 720p ][ 28 1.679300 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 33 1.895006 Star Trek [2009] [1080p] ][ 35 7.705054 Ice Age 1 [2002] [720p] ][ 41 3.812025 Chicken Run [2000] [720p] ][ 44 7.472310 Duplicity 2009   720p ][ 48 2.112130 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 54 8.624808 Connected   720p ][ 69 5.218473 Terminator 2 Judgement Day Directors Cut [1991]   1080p ]
[ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 13 2.818495 Doomsday 720p ][ 16 3.041106 iRobot  720p ][ 21 5.124632 Monsters vs Aliens [2009] [720p] ][ 24 5.036194 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 28 1.679300 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 29 6.672017 Star Trek Collection 1-9    720p ][ 30 3.660891 James Bond 03 - GoldFinger [1964] ][ 33 1.895006 Star Trek [2009] [1080p] ][ 41 3.812025 Chicken Run [2000] [720p] ][ 44 7.472310 Duplicity 2009   720p ][ 47 0.322202 Chicken Little 1080p ][ 49 4.041338 Dragonball Evolution [2009] [720p] ][ 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 54 8.624808 Connected   720p ][ 59 7.119242 Jurassic Park 1-3  1080p ][ 69 5.218473 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 78 8.245003 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ]
[ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 13 2.818495 Doomsday 720p ][ 21 5.124632 Monsters vs Aliens [2009] [720p] ][ 24 5.036194 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 28 1.679300 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 29 6.672017 Star Trek Collection 1-9    720p ][ 30 3.660891 James Bond 03 - GoldFinger [1964] ][ 31 5.085944 Big Nothing   720p ][ 33 1.895006 Star Trek [2009] [1080p] ][ 41 3.812025 Chicken Run [2000] [720p] ][ 42 1.063909 007 Series ][ 44 7.472310 Duplicity 2009   720p ][ 47 0.322202 Chicken Little 1080p ][ 49 4.041338 Dragonball Evolution [2009] [720p] ][ 52 1.900196 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 54 8.624808 Connected   720p ][ 59 7.119242 Jurassic Park 1-3  1080p ][ 60 5.062544 The Matrix   1080p ][ 69 5.218473 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 71 0.744343 The Usual Suspect    720p ][ 78 8.245003 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 80 8.351246 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 85 5.971929 The Chronicle of Riddick 720p ]
[ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 7 9.025733 X-Men Origin - Wolverine [2009] [1080p] ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 13 2.818495 Doomsday 720p ][ 20 6.324520 Underworld Series ][ 21 5.124632 Monsters vs Aliens [2009] [720p] ][ 23 4.299087 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 28 1.679300 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 29 6.672017 Star Trek Collection 1-9    720p ][ 30 3.660891 James Bond 03 - GoldFinger [1964] ][ 31 5.085944 Big Nothing   720p ][ 33 1.895006 Star Trek [2009] [1080p] ][ 41 3.812025 Chicken Run [2000] [720p] ][ 42 1.063909 007 Series ][ 44 7.472310 Duplicity 2009   720p ][ 47 0.322202 Chicken Little 1080p ][ 49 4.041338 Dragonball Evolution [2009] [720p] ][ 54 8.624808 Connected   720p ][ 56 5.219541 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 57 6.903327 Burn E [720p] ][ 59 7.119242 Jurassic Park 1-3  1080p ][ 60 5.062544 The Matrix   1080p ][ 70 2.588273 Battle for Terra [2007] [1080p] ][ 71 0.744343 The Usual Suspect    720p ][ 75 8.039718 Big Nothing   720p ][ 78 8.245003 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 80 8.351246 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 85 5.971929 The Chronicle of Riddick 720p ][ 94 4.089340 Night At The Museum 2 [720p] ]
[ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 20 6.324520 Underworld Series ][ 21 5.124632 Monsters vs Aliens [2009] [720p] ][ 23 4.299087 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 29 6.672017 Star Trek Collection 1-9    720p ][ 30 3.660891 James Bond 03 - GoldFinger [1964] ][ 31 5.085944 Big Nothing   720p ][ 33 1.895006 Star Trek [2009] [1080p] ][ 41 3.812025 Chicken Run [2000] [720p] ][ 42 1.063909 007 Series ][ 44 7.472310 Duplicity 2009   720p ][ 46 1.234344 Die Hard 3   720p ][ 47 0.322202 Chicken Little 1080p ][ 49 4.041338 Dragonball Evolution [2009] [720p] ][ 53 8.721944 A Bug's Life [720p] ][ 54 8.624808 Connected   720p ][ 56 5.219541 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 59 7.119242 Jurassic Park 1-3  1080p ][ 60 5.062544 The Matrix   1080p ][ 70 2.588273 Battle for Terra [2007] [1080p] ][ 71 0.744343 The Usual Suspect    720p ][ 75 8.039718 Big Nothing   720p ][ 78 8.245003 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 80 8.351246 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 81 2.780032 City of Embers    720p ][ 82 5.081608 Bridge To Terabithia 720P ][ 84 6.959754 Terminator 3 Rise Of The Machine [2003]   720p ][ 85 5.971929 The Chronicle of Riddick 720p ][ 91 2.122059 American Psycho   720p ][ 92 0.945333 Sixth Sense   720p ][ 93 5.511501 Hellboy 2     1080p ][ 94 4.089340 Night At The Museum 2 [720p] ][ 103 5.516839 Hulk 2 - The Incredible Hulk - 720p ][ 110 4.048022 The Matrix Revolution   1080p ]
[ 2 9.730302 007 Thunderball [1965]  [BluRay] [720p] [DTS] ][ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 20 6.324520 Underworld Series ][ 21 5.124632 Monsters vs Aliens [2009] [720p] ][ 23 4.299087 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 27 2.160895 Orphan (2009)    1080p ][ 29 6.672017 Star Trek Collection 1-9    720p ][ 31 5.085944 Big Nothing   720p ][ 33 1.895006 Star Trek [2009] [1080p] ][ 41 3.812025 Chicken Run [2000] [720p] ][ 42 1.063909 007 Series ][ 44 7.472310 Duplicity 2009   720p ][ 46 1.234344 Die Hard 3   720p ][ 47 0.322202 Chicken Little 1080p ][ 49 4.041338 Dragonball Evolution [2009] [720p] ][ 53 8.721944 A Bug's Life [720p] ][ 54 8.624808 Connected   720p ][ 56 5.219541 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 59 7.119242 Jurassic Park 1-3  1080p ][ 60 5.062544 The Matrix   1080p ][ 63 1.817112 Aeon Flux   720p ][ 64 2.923004 Hulk 2 - The Incredible Hulk - 720p ][ 70 2.588273 Battle for Terra [2007] [1080p] ][ 71 0.744343 The Usual Suspect    720p ][ 75 8.039718 Big Nothing   720p ][ 76 3.351120 The Day After Tomorrow 1080p ][ 80 8.351246 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 81 2.780032 City of Embers    720p ][ 82 5.081608 Bridge To Terabithia 720P ][ 84 6.959754 Terminator 3 Rise Of The Machine [2003]   720p ][ 85 5.971929 The Chronicle of Riddick 720p ][ 88 4.953958 Jeniffer's Body ( 2009)    1080p ][ 91 2.122059 American Psycho   720p ][ 92 0.945333 Sixth Sense   720p ][ 93 5.511501 Hellboy 2     1080p ][ 94 4.089340 Night At The Museum 2 [720p] ][ 96 9.052782 Final Fantasy The Spirits Within    720p ][ 103 5.516839 Hulk 2 - The Incredible Hulk - 720p ][ 104 0.841493 Taken 1080p ][ 110 4.048022 The Matrix Revolution   1080p ]
[ 2 9.730302 007 Thunderball [1965]  [BluRay] [720p] [DTS] ][ 3 3.531948 Horton Hears a Who! [2008] [720p] ][ 5 9.728215 Body of Lies   720p ][ 8 5.011611 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 20 6.324520 Underworld Series ][ 21 5.124632 Monsters vs Aliens [2009] [720p] ][ 22 4.446526 X-Men 3   1080p ][ 23 4.299087 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 27 2.160895 Orphan (2009)    1080p ][ 41 3.812025 Chicken Run [2000] [720p] ][ 42 1.063909 007 Series ][ 44 7.472310 Duplicity 2009   720p ][ 46 1.234344 Die Hard 3   720p ][ 47 0.322202 Chicken Little 1080p ][ 49 4.041338 Dragonball Evolution [2009] [720p] ][ 50 8.118501 Indiana.Jones.and.the.Kingdom.of.the.Crystal.Skull    1080p ][ 51 4.549279 Lion King, The 720p ][ 53 8.721944 A Bug's Life [720p] ][ 59 7.119242 Jurassic Park 1-3  1080p ][ 60 5.062544 The Matrix   1080p ][ 61 1.298710 Eragon [2006] [720p] ][ 63 1.817112 Aeon Flux   720p ][ 64 2.923004 Hulk 2 - The Incredible Hulk - 720p ][ 67 8.151021 IMAX Ring of Fire [720p] [Documenter] ][ 70 2.588273 Battle for Terra [2007] [1080p] ][ 71 0.744343 The Usual Suspect    720p ][ 75 8.039718 Big Nothing   720p ][ 76 3.351120 The Day After Tomorrow 1080p ][ 80 8.351246 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 81 2.780032 City of Embers    720p ][ 82 5.081608 Bridge To Terabithia 720P ][ 84 6.959754 Terminator 3 Rise Of The Machine [2003]   720p ][ 85 5.971929 The Chronicle of Riddick 720p ][ 86 4.615148 Kung Fu Panda (BluRay 720p) ][ 88 4.953958 Jeniffer's Body ( 2009)    1080p ][ 92 0.945333 Sixth Sense   720p ][ 93 5.511501 Hellboy 2     1080p ][ 94 4.089340 Night At The Museum 2 [720p] ][ 95 9.559426 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 96 9.052782 Final Fantasy The Spirits Within    720p ][ 98 3.501873 Madagascar 1 [2005] [1080p] ][ 99 7.994218 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 103 5.516839 Hulk 2 - The Incredible Hulk - 720p ][ 104 0.841493 Taken 1080p ][ 110 4.048022 The Matrix Revolution   1080p ][ 112 7.376703 Rocky 5 [1990] [BluRay] [720p] [AC3] ][ 117 4.067580 James Bond 08 - Live and Let Die [1973] ][ 121 0.971639 Eragon 720p ][ 124 6.319414 Iron Man    1080p ]
